### ISSUE TITLE
ChakraCore 1.11.5 fails to verify checksum because it expects lowercase string

### ISSUE THREAD

**Issue by akoeplinger** [GoogleChromeLabs/jsvu#54]
The regex in https://github.com/GoogleChromeLabs/jsvu/blob/5da9253b29b813f1799368432cd4dfee2f3c46f7/engines/chakra/get-checksums.js#L21 expects lowercase `a-f` but the latest ChakraCore release used uppercase chars in their checksum string (https://github.com/Microsoft/ChakraCore/issues/5902).

Would it be possible to modify the regex to allow uppercase `A-F` too?

**Comment by mathiasbynens** [comment]
We can hotfix jsvu for this case.

It would be sweet if the Chakra team could automate this part of their release process (the variation in output as well as the recent [incorrect checksum](https://github.com/GoogleChromeLabs/jsvu/issues/17) implies this hasn’t been done yet) and stick to a single particular format, though. cc @dilijev @obastemur

**Comment by mathiasbynens** [comment]
It would be even better to provide a JSON file at a predictable URL with the latest version number + checksum in it.

(This is what V8 does.)

**Comment by akoeplinger** [comment]
@mathiasbynens it looks like there's another issue, the string comparison needs to be done case insensitively too:

```
17:34:10 📦 jsvu v1.3.2 — the JavaScript engine Version Updater 📦
17:34:10 ✔ Read OS from config: linux64
17:34:10 ✔ Read engines from config: chakra, javascriptcore, spidermonkey, v8, xs
17:34:10 ❯ Finding the latest Chakra version…
17:34:10 [1A[K✔ Found latest Chakra version: v1.11.5.
17:34:10 ❯ Predicting URL…
17:34:10 [1A[K✔ URL: https://aka.ms/chakracore/cc_linux_x64_1_11_5
17:34:10 ❯ Getting SHA-256 checksum…
17:34:11 [1A[K✔ SHA-256 checksum: 043E161E583C2134DB890CDCF4C427A77AD023C13C966E390307A7E9746A8AC1
17:34:11 ❯ Downloading and verifying checksum…
17:34:15 
17:34:15 [1A[2K[1A
17:34:15 ✖ Invalid checksum. Expected: 043E161E583C2134DB890CDCF4C427A77AD023C13C966E390307A7E9746A8AC1 Actual: 043e161e583c2134db890cdcf4c427a77ad023c13c966e390307a7e9746a8ac1
```

**Comment by mathiasbynens** [comment]
_facepalm_ Properly fixed in jsvu@1.3.3!

**Comment by akoeplinger** [comment]
thanks, all good now ❤️
